.github/workflows/coverage.yml

```diff
@@ -90,6 +90,7 @@ jobs:
     runs-on:
       labels: bare-metal
     needs: ["set-tags"]
+    if: ${{ !github.event.pull_request.head.repo.fork }}
```

Comment:
We do need the set-tags step for PRs coming from forks, all the jobs depend on set-tags. Or am I missing something?

Reply:
nevermind, I got it now